frontend/helpers/issues.ts

```diff
@@ -15,34 +15,50 @@
  *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-import { get, omit } from 'lodash';
+import { get } from 'lodash';
 import { getAPIUrl } from '../services/api';
 import { STATUSES_COLOURS, STATUSES_ICONS, STATUSES } from '../constants/issues';
 import { isAdmin, hasPermissions, PERMISSIONS } from './permissions';
-import { sortByDate } from './sorting';
+
+const renameFieldIfExists = (issue, fieldName, newFieldName) => {
```

Comment:
If you want to do that this way, it should be a pure function.

e.g
```suggestion
const renameFieldIfExists = (issue, fieldName, newFieldName) => {
   const updatedIssue = { ...issue };
   updatedIssue[newFieldName] = issue[fieldName];
   delete updatedIssue[fieldName];
   return updatedIssue;
}
```

Reply:
that would be less performant. I'll stick to your comments that makes this code more performant.